src/completion.ts

```diff
@@ -14,7 +14,9 @@ export class HashtagCompletionItemProvider implements vscode.CompletionItemProvi
       .split("")
       .reverse()
       .join("")
-      .match(/^\/[^\s!@#$%^&*()=+.,\[{\]};:'"?><]+#(\s|$)/);
+      // #24 in addition to "/" also "." can be used as tag separator
+      // .match(/^\/[^\s!@#$%^&*()=+.,\[{\]};:'"?><]+#(\s|$)/);
+      .match(/^\/[^\s!@#$%^&*()=+,\[{\]};:'"?><]+#(\s|$)/);
```

Comment:
Can we moved the regexp to hashtagRegexp from parser?

Reply:
maybe - I just modified, what I found in the existing code.
And what I changed here may not work, because it needs to be changed in combination with line 39 and line 62
If you have an idea, what and how to move - fine!

In General it is a good idea to the logic only once. But they are a bit differnt, and I did not explore or evaluate, why:

```
  .match(/^\/[^\s!@#$%^&*()=+,\[{\]};:'"?><]+#(\s|$)/);
/(?<=(^|\s)#)[^\s!@#$%^&*()=+,\[{\]};:'"?><]+[\s]?/g;
```